x/gov/handler.go

```diff
@@ -114,8 +114,8 @@ func EndBlocker(ctx sdk.Context, keeper Keeper) (resTags sdk.Tags) {
 		resTags.AppendTag(tags.Action, tags.ActionProposalDropped)
 		resTags.AppendTag(tags.ProposalID, proposalIDBytes)
 
-		logger.Info(fmt.Sprintf("Proposal %d - \"%s\" - didn't mean minimum deposit (had only %s), deleted",
-			inactiveProposal.GetProposalID(), inactiveProposal.GetTitle(), inactiveProposal.GetTotalDeposit()))
+		logger.Info(fmt.Sprintf("proposal %d (%s) didn't meet minimum deposit of %v steak (had only %s steak); deleted",
```

Comment:
Great! But its not always going to be `steak` is it? Is there a way to get the denomination? Also this line is extremely long. I recommend breaking it up:

```
logger.Info(
    fmt.Sprintf("proposal %d (%s) didn't meet minimum deposit of %v steak (had only %s steak); deleted",
        inactiveProposal.GetProposalID(),
        inactiveProposal.GetTitle(),
        keeper.GetDepositProcedure(ctx).MinDeposit.AmountOf("steak"),
        inactiveProposal.GetTotalDeposit().AmountOf("steak"),
    ),
)
```

Reply:
Hrmm, not using Steak here might be hard, since MinDeposit is `sdk.Coins`. This confuses me though, shouldn't it just be the staking token (and therefore `sdk.Coin`)

/cc @sunnya97

Reply:
We should have a `const` with the coin denomination of the governance deposit and just use it everywhere in the module

Reply:
Should not be a constant, it should be obtained from the global param store once that is in. I think we can leave this as-is and make a follow-up issue to derive all "steak" from global param store once Joon's PR's are merged.

Reply:
Good points @ValarDragon